docs/supply-chain-tools/keystore-guide.md

```diff
@@ -26,11 +26,11 @@ Choose the appropriate method for your situation. This process must be done for
 
 ## Step 1: Key Store File Initialization
 
-Use the **p12init.sh** script provided along with the toolkit to generate the key store file. Before use, update the configuration in the file (For example, key store password and certificate validity period).
+Use the [p12init.sh](https://github.com/secure-device-onboard/supply-chain-tools/blob/v1.9.0/scripts/p12init.sh) script provided along with the toolkit to generate the key store file. Before use, update the configuration in the file (For example, key store password and certificate validity period).
```

Comment:
Use the master branch notation, as otherwise we have to keep updating this for every release.

Reply:
Done